drivers/pulse_counter/include/pulse_counter_params.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2017 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_pulse_counter
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for PULSE_COUNTER devices
+ *
+ * @author      Hyung-Sin Kim <hs.kim@cs.berkeley.edu>
+ */
+
+#ifndef PULSE_COUNTER_PARAMS_H
+#define PULSE_COUNTER_PARAMS_H
+
+#include "board.h"
+#include "pulse_counter.h"
+#include "saul_reg.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Set default configuration parameters for the PULSE_COUNTER driver
+ * @{
+ */
+#ifndef PULSE_COUNTER_PARAMS_BOARD
+#define PULSE_COUNTER_PARAMS_DEFAULT  { .gpio = GPIO_PIN(PA, 19) }
```

Comment:
PA is CPU specific, I don't think there is any sensible default possible here (similar to how there is no default gpio_params.h)

Reply:
I agree. I will remove DEFAULT parameter so that RIOT incurs compile error without defining BOARD_PARAMS.